src/Microsoft.AspNet.Mvc/MvcServiceCollectionExtensions.cs

```diff
@@ -22,7 +22,7 @@ private static void ConfigureDefaultServices(IServiceCollection services, IConfi
             services.AddOptions(configuration);
             services.AddDataProtection(configuration);
             services.AddRouting(configuration);
-            services.AddContextAccessor(configuration);
+            services.AddScopeLocal(configuration);
```

Comment:
Am I reading this correctly and AddMvc() stick the IConfiguration into DI? I thought that was not supposed to happen.